crates/forge/bin/cmd/selectors.rs

```diff
@@ -36,6 +36,7 @@ pub enum SelectorsSubcommands {
     #[command(visible_alias = "up")]
     Upload {
         /// The name of the contract to upload selectors for.
+        /// Can also be in form of `path:contract name`.
         #[arg(required_unless_present = "all")]
         contract: Option<String>,
```

Comment:
do we not have a helper type for this? path or path:contract

Reply:
yep, missed that one, reused in https://github.com/foundry-rs/foundry/pull/10073/commits/d64606aaf8d0be66247ee9b9196b4cb9a5381939